src/main/java/frc/robot/subsystems/Hood.java

```diff
@@ -47,6 +47,14 @@ public double getMeasurement() {
     return getAngle();
   }
 
+  /**
+   * Gets whether the hood is at the desired angle
+   * @return true if the hood is at the desired angle
```

Comment:
Nitpick: You don't need this duplication for simple javadocs

Reply:
resolved in 1e533bc